workflow/rocoto/gfs_tasks.py

```diff
@@ -553,17 +553,31 @@ def _fcst_cycled(self):
 
     def post(self):
         add_anl_to_post = False
-        if self.app_config.mode in ['cycled']:
-            add_anl_to_post = True
-
         return self._post_task('post', add_anl_to_post=add_anl_to_post)
 
+    def postanl(self):
+        postenvars = self.envars.copy()
+        for key, value in postenvar_dict.items():
+            postenvars.append(rocoto.create_envar(name=key, value=str(value)))
+
+        deps = []
+        atm_hist_path = self._template_to_rocoto_cycstring(self._base["COM_ATMOS_HISTORY_TMPL"])
+        data = f'{atm_hist_path}/{self.cdump}.t@Hz.atm.loganl.txt'
+        dep_dict = {'type': 'data', 'data': data}
+        deps.append(rocoto.add_dependency(dep_dict))
+        dependencies = rocoto.create_dependency(dep=deps)
+        resources = self.get_resource('post')
+        task = create_wf_task('postanl', resources, cdump=self.cdump, envar=postenvars, dependency=dependencies,
+                              cycledef=self.cdump)
+
+        return task
+
     def ocnpost(self):
         if self.app_config.mode in ['forecast-only']:  # TODO: fix ocnpost in cycled mode
             return self._post_task('ocnpost', add_anl_to_post=False)
 
     def _post_task(self, task_name, add_anl_to_post=False):
-        if task_name not in ['post', 'ocnpost']:
+        if task_name not in ['postanl', 'post', 'ocnpost']:
```

Comment:
Please remove argument `add_anl_to_post`.
Please remove lines 583, 584.

Reply:
Also on line 577

Reply:
`postanl` does not belong here.  
```suggestion
        if task_name not in ['post', 'ocnpost']:
```